### PR TITLE
Add multi-env configuration

### PR DESCRIPTION
**Pull request by meibensteiner**
This is an additional example for customers who want to route the logs of certain namespaces to different dynatrace log endpoints.

Replaces #35.

### CODE REVIEW

File: example/README.md [hunk 1]
```diff
@@ -80,3 +80,17 @@ Build the FluentD docker image provided in our example and upload it to your rep
 3. Deploy `fluentd.yaml`
 
    ``kubectl apply -f fluentd.yaml``
+
+## Sending logs to different Dynatrace environments
+
+With the introduction of Cloud Native Full Stack injection [example](https://github.com/Dynatrace/dynatrace-operator/blob/master/config/samples/cloudNativeFullStack.yaml) it is possible to send metrics and traces to more than one Dynatrace environment. The same thing can be done for logs using fluentd. The following example serves as a potential reference implementation. Results may vary.
```
Comment (dyladan) [inline example/README.md:86]: To me, it is confusing that the prose doesn't mention you're talking about a feature of the Dynatrace operator. It sounds like a kubernetes feature or a feature of this repo.

File: example/README.md [hunk 2]
```diff
@@ -80,3 +80,17 @@ Build the FluentD docker image provided in our example and upload it to your rep
...
+
+Adapt the "option 2" instructions above as follows:
+
+1. Create a second API token, from a second Dynatrace environment. Create a second kubernetes for it as well (ex: `log-ingest2`)
```
Comment (dyladan) [inline example/README.md:90]: A whole second kubernetes cluster? Isn't this about sending to multiple environments from a single cluster?

File: example/README.md [hunk 3]
```diff
@@ -80,3 +80,17 @@ Build the FluentD docker image provided in our example and upload it to your rep
...
+
```
Comment (dyladan) [inline example/README.md:91]: Are the blank lines between bullets intentional?

File: example/README.md [hunk 4]
```diff
@@ -80,3 +80,17 @@ Build the FluentD docker image provided in our example and upload it to your rep
...
+2. Adapt the fluentd config map to include a second API endpoints, and a filter for targeted namespaces as shown in [configmap-multi.yaml](configmap-multi.yaml).
+
+3. Adapt the fluentd.yaml file to reference both API endpoints (`INGEST_ENDPOINT1 and INGEST_ENDPOINT2`) as shown in [fluentd-multi.yaml](fluentd-multi.yaml).
```
Comment (dyladan) [inline example/README.md:94]: It's easy to miss the 1 and 2 here. Maybe separate them with an underscore to make them more obvious like `INGEST_ENDPOINT_1`?

File: example/configmap-multi.yaml [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+      api_token "#{ENV['LOG_INGEST_TOKEN2']}"
+      ssl_verify_none true
+      <buffer>
+        retry_max_times 3
+      </buffer>
+    </match>
+
+    <system>
+        log_level info
+    </system>
```
Comment (dyladan) [inline example/configmap-multi.yaml:112]: style nit: missing trailing newline

File: example/fluentd-multi.yaml [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+      - name: var-log-pos
+        hostPath:
+          path: /var/log/fluentd.pos
+          type: FileOrCreate
+      - name: fluentd-ingest-configuration
+        configMap:
+          name: fluentd-ingest-configuration
+          items:
+          - key: fluent.conf
+            path: fluent.conf
```
Comment (dyladan) [inline example/fluentd-multi.yaml:101]: style nit: missing trailing newline

**Comment by meibensteiner** [comment]
Issues were addressed.

File: example/README.md [hunk 7]
```diff
@@ -76,7 +69,17 @@ Build the FluentD docker image provided in our example and upload it to your rep
...
 
    ``kubectl apply -f fluentd.yaml``
+
+## Sending logs to different Dynatrace environments
+
+With the introduction of [Cloud Native Full Stack injection](https://github.com/Dynatrace/dynatrace-operator/blob/master/config/samples/cloudNativeFullStack.yaml), a feature of our [Dynatrace Operator](https://github.com/Dynatrace/dynatrace-operator/), it is possible to send metrics and traces to more than one Dynatrace environment. The same thing can be done for logs using fluentd. The following example serves as a potential reference implementation. Results may vary.
+
+Adapt the "option 2" instructions above as follows:
+
+1. Create a second API token, from a second Dynatrace environment. Create a second kubernetes secret for it as well (ex: `log-ingest2`)
```
Comment (dyladan) [inline example/README.md:82]: A second secret makes a lot more sense :)

**Review by dyladan — APPROVED** [review]
Looks great thanks for the contribution

**Pull request merged**